administrator/components/com_modules/src/Model/ModulesModel.php

```diff
@@ -312,6 +344,24 @@ protected function getListQuery()
 		$query->select($db->quoteName('e.name', 'name'))
 			->join('LEFT', $db->quoteName('#__extensions', 'e') . ' ON ' . $db->quoteName('e.element') . ' = ' . $db->quoteName('a.module'));
 
+		// Join over the workflows association
+		$query->select($db->quoteName('wa.stage_id', 'stage_id'))
+			->join('INNER', $db->quoteName('#__workflow_associations', 'wa'), $db->quoteName('wa.item_id') . ' = ' . $db->quoteName('a.id'))
+			->where($db->quoteName('wa.extension') . ' = ' . $db->quote('com_modules.module');
```

Comment:
```suggestion
			->where($db->quoteName('wa.extension') . ' = ' . $db->quote('com_modules.module'));
```
Sorry, was my mistake in my suggestion